src/SFA.DAS.RoatpGateway.Web/Startup.cs

```diff
@@ -218,13 +218,14 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             }
 
             app.UseHttpsRedirection();
-            app.UseStaticFiles();
             app.UseCookiePolicy();
-            app.UseAuthentication();
             app.UseSession();
             app.UseRequestLocalization();
             app.UseStatusCodePagesWithReExecute("/ErrorPage/{0}");
             app.UseSecurityHeaders();
+            app.UseMiddleware<SecurityHeadersMiddleware>();
```

Comment:
At the risk of being picky ;-) I would just keep it consistent by using `Use` method, I doubt there is any difference. Just would like to stay consistent in our approach.

Reply:
ok done